orbit/pkg/packaging/macos_templates.go

```diff
@@ -55,6 +55,8 @@ ln -sf /opt/orbit /var/lib/orbit
 DAEMON_LABEL="com.fleetdm.orbit"
 DAEMON_PLIST="/Library/LaunchDaemons/${DAEMON_LABEL}.plist"
 
+# Stop the previous desktop agent
+pkill fleet-desktop
```

Comment:
We might need to do `pkill fleet-desktop || true` here?

To not fail the script if it's not running:
```sh
$ pkill foo
$ echo $?
1
```

Reply:
There's no `set -e` so it shouldn't be technically necessary but that is cleaner